nustar_pysolar/map.py

```diff
@@ -94,18 +104,31 @@ def make_sunpy(evtdata, hdr, norm_map=False):
     "CUNIT2": "arcsec",
     "CTYPE2": "HPLT-TAN",
     "PIXLUNIT": pixluname,
-    "HGLT_OBS": get_sun_B0(mid_obs_time),
+    "DETECTOR":"NuSTAR",
+#    SunPy v1+ version
+    "HGLT_OBS": sunpy.coordinates.sun.B0(mid_obs_time),
+#    SunPy <v1 version        
+#   "HGLT_OBS": sunpy.coordinates.get_sun_B0(mid_obs_time),
     "HGLN_OBS": 0,
-    "RSUN_OBS": sun.solar_semidiameter_angular_size(mid_obs_time).value,
-    "RSUN_REF": sun.constants.radius.value,
+#    SunPy v1+ version
+    "RSUN_OBS": sunpy.coordinates.sun.angular_radius(mid_obs_time).value,
+#    SunPy <v1 version
+#    "RSUN_OBS": sun.solar_semidiameter_angular_size(mid_obs_time).value,
+##        
+#    sun.constants will stay in SunPy v1+ ? just sun.sun being depreciated?
+#      https://docs.sunpy.org/en/stable/code_ref/sun.html#module-sunpy.sun.models
+    "RSUN_REF": sunpy.sun.constants.radius.value,
+###       
     # Assumes dsun_obs in m if don't specify the units, so give units
-    "DSUN_OBS": get_sunearth_distance(mid_obs_time).value*u.astrophys.au
+#    SunPy v1+ version        
+    "DSUN_OBS": sunpy.coordinates.sun.earth_distance(mid_obs_time).value*u.astrophys.au
```

Comment:
```suggestion
    "DSUN_OBS": sunpy.coordinates.sun.earth_distance(mid_obs_time).to('AU')
```
Alternatively, if having `DSUN_OBS` as a number with units causes any issues, you can instead just get its value in meters: `sunpy.coordinates.sun.earth_distance(mid_obs_time).to_value('m')`.